src/lib/connectors/repos/queries/gauging-stations.js

```diff
@@ -0,0 +1,15 @@
+exports.findGaugingStations = `select 
+water.licence_version_purpose_condition_types.*,
+water.licence_version_purpose_conditions.*,
+water.licence_version_purposes.*,
+water.licence_versions.end_date,
+water.gauging_stations.* from 
+water.gauging_stations join water.gauging_station_condition on (water.gauging_stations.gauging_station_id = water.gauging_station_condition.gauging_station_id)  
+join water.licences on (water.gauging_station_condition.licence_id = water.licences.licence_id)
+join water.licence_versions on (water.licences.licence_id = licence_versions.licence_id)
+join water.licence_version_purposes on (water.licence_versions.licence_version_id = water.licence_version_purposes.licence_version_id)
+join water.licence_version_purpose_conditions on (water.licence_version_purposes.licence_version_purpose_id = water.licence_version_purpose_conditions.licence_version_purpose_id) 
+join water.licence_version_purpose_condition_types on (water.licence_version_purpose_conditions.licence_version_purpose_condition_type_id =
+water.licence_version_purpose_condition_types.licence_version_purpose_condition_type_id) 
+where (water.licence_versions.end_date is null or water.licence_versions.end_date > now())
+and water.gauging_stations.gauging_station_id=:gaugingStationId;`;
```

Comment:
Can you also add a few more conditions please:

1. Check that the licence is not revoked with `and (water.licences.revoked_date is null or water.licences.revoked_date > now())`
2. Check that the licence is not expired with `and (water.licences.expired_date is null or water.licences.expired_date > now())`
3. Check that the licence is not lapsed with `and (water.licences.lapsed_date is null or water.licences.lapsed_date > now())`
4. Check that the linkage has not been deleted with `and (water.licence_gauging_stations.date_deleted is null or water.licence_gauging_stations.date_deleted > now())